package.json

```diff
@@ -13,7 +13,8 @@
   "main": "./base",
   "scripts": {
     "prebuild": "yarn run build:clean",
-    "build:clean": "rimraf build rules customRules base.js react.js untyped.js",
+    "clean": "rimraf build rules customRules '*.js'",
```

Comment:
Typically our scripts without namespaces combine operations, e.g.,

`"clean": "yarn run clean:foo && yarn run clean:bar"`

Is there a good reason to invert the convention in this file?